3rdparty/expreval/docs/source.txt

```diff
@@ -0,0 +1 @@
+Source of the library can be found at https://expreval.sourceforge.net
```

Comment:
Here also, would be nice to state which version was imported. I assume it is version 3.5 which is the last update available on that site (from April 2008, before IFEM was born).

Reply:
Changelog mentions 3.6 though so it's likely some pre-3.6 version taken from cvs at that point in time.